library/core-library/src/main/java/uk/gov/gchq/gaffer/function/aggregate/ArrayListConcat.java

```diff
@@ -13,19 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package uk.gov.gchq.gaffer.function.simple.aggregate;
+package uk.gov.gchq.gaffer.function.aggregate;
 
 import org.apache.commons.lang.builder.EqualsBuilder;
 import org.apache.commons.lang.builder.HashCodeBuilder;
 import org.apache.commons.lang.builder.ToStringBuilder;
 import uk.gov.gchq.gaffer.function.SimpleAggregateFunction;
 import uk.gov.gchq.gaffer.function.annotation.Inputs;
 import uk.gov.gchq.gaffer.function.annotation.Outputs;
+
 import java.util.ArrayList;
 
 /**
  * An <code>ArrayListConcat</code> is a {@link SimpleAggregateFunction} that concatenates
- * {@link java.util.ArrayList}s together.
```

Comment:
Do we want to move away from fully qualified references in all the Javadoc comments?